wg-structured-logging/annual-report-2022.md

```diff
@@ -5,56 +5,53 @@
 1. What work did the WG do this year that should be highlighted?
    For example, artifacts, reports, white papers produced this year.
 
-   -
-   -
-   -
+   - Graduated [Deprecation of klog specific flags](https://github.com/kubernetes/enhancements/issues/2845) to GA
+   - Added [Contextual logging](https://github.com/kubernetes/enhancements/issues/3077) as Alpha
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
+   - Continuously migrating kubernetes/kubernetes repository to structured, contextual logging.
+     In 2022 and early 2023, 82% of kube-controller-manager were converted, over 1500 log calls.
+   - Maintaining and enhancing klog. In 2022, [performance improved](https://github.com/kubernetes/kubernetes/pull/115277)
+     between up to 45% depending on the component and output format.
+   - Maintaining [logcheck](https://github.com/kubernetes-sigs/logtools/tree/main/logcheck).
+     In 2022, it was enhanced and moved into a new repository.
```

Comment:
The answers to these two sections should be swapped.

Reply:
I'm not sure what exactly I should swap.

I can see how "migrating kubernetes/kubernetes" could fit under "What work did the WG do this year that should be highlighted?".

But the two points there (deprecation and contextual logging KEP) don't fit under "initiatives ... that *aren't* being tracked in KEPs" (emphasis mine)?

Reply:
Ah sorry. I misinterpreted the first question since it differs from the SIG Annual Report template.

We are good to go here.